src/content/docs/ts-sdk/outputs/rtp.mdx

```diff
@@ -313,6 +316,33 @@ Audio output encoder preset.
     - `voip` - Recommended for VoIP and videoconferencing applications, prioritizing listening quality and speech intelligibility.
     - `lowest_latency` - Recommended **only** when achieving the lowest possible latency is the highest priority.
 
+--- 
+
+#### sampleRate
+Sample rate.
+
+- **Type**: `u32`
+- **Default value**: `48000`
+- **Supported values**: `8000`, `16000`, `24000`, `48000`
+
+---
+
+#### forwardErrorCorrection
+Specifies whether the stream uses forward error correction. It’s specific for the `Opus` codec. For more information, visit RFC specification sections [2.1.7](https://datatracker.ietf.org/doc/html/rfc6716#section-2.1.7)
+and [4.2.5](https://datatracker.ietf.org/doc/html/rfc6716#section-4.2.5)
+
+- **Type**: `bool`
+- **Default value**: `false`
+
+---
+
+#### expectedPacketLoss
+Expected approximate packet loss percentage. When forward error correction is enabled this value should be greater than 0.
```

Comment:
"Expected approximate" does not sound natural. If you are talking about packet loss "approximate" is kind of implied.

Also, I think this is a good place to explain how this value is used. e.g. "This value controls how much error correction information wll be included in the stream"